gp-cli.md

```diff
@@ -0,0 +1,344 @@
+# Globalization Pipeline CLI Tool User Guide
+<!--
+/*  
+ * Copyright IBM Corp. 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-->
+
+* [Overview](#TOC-Overview)
+* [Prerequisites](#TOC-Prerequisites)
+* [Command Reference](#TOC-Command-Reference)
+  * [Help Command](#TOC-Cmd-Help)
+  * [Bundle Commands](#TOC-Cmd-Bundle)
+  * [User Commands](#TOC-Cmd-User)
+  * [Other Commands](#TOC-Cmd-Others)
+* [How-To](#TOC-How-To)
+  * [Import Existing Translations](#TOC-Import-Existing-Translations)
+
+---
+## <a name="TOC-Overview"></a>Overview
+
+Globalization Pipeline CLI (Command Line Interface) Tool is designed for
+manipulating translation bundles hosted by Globalization Pipeline service
+on command line.
+
+---
+## <a name="TOC-Prerequisites"></a>Prerequisites
+
+Globalization Pipeline CLI Tool is distributed in a single jar package.
+You need Java SE Runtime Environment 7 or later version to run the tool.
+
+---
+## <a name="TOC-Command-Reference"></a>Command Reference
+
+Most of commands takes user credentials for a Globalization Pipeline service
+instance. The common command options for specifying user credentials are
+below.
+
+* -s (--serviceUrl): Globalization Pipeline service URL
+* -i (--instanceId): Service instance ID
+* -u (--user): User ID
+* -p (--password): Password
+
+For example,
+```
+java -jar gp-cli.jar list-bundle -s https://gp-rest.ng.bluemix.net/translate/rest -i 1d42b9329d6f5ab173c82810c819afa6 -u 8492e8d7bc28d9dc34a31a6d0ec7384e -p DoRfuq1w1ohx8vKXoZHHMenxxpSyoF0u
+```
+You can access these credentials on Bluemix Dashboard. For more details,
+plese see [Credentials](https://github.com/IBM-Bluemix/gp-common/blob/master/README.md#4-credentials)
+section in the Globalization Pipeline Quick Start Guide.
+
+Alternatively, you can store the credentials in a JSON file and use -j (--jsonCreds)
+option to specify the file. For example, you can create a JSON file `mycreds.json'
```

Comment:
minor: missing trailing backtick (`) — in general I would probably quote anything on the console, such as `--jsonCreds`